docs/1.introduction/1.index.mdx

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
When a commit moves or renames files, don't do any change on the contents of the files within the same commit. Otherwise, on the PR it's too difficult to spot the differences because github is sometimes not able to put side by side the old and the new file.